docs/dev/copyright_license.md

```diff
@@ -0,0 +1,206 @@
+- If your software includes other code from the RST (or other GPL-licensed code), your software can only be licensed under GPL. 
+
+
+__Am I allowed to copy code from the RST into my own software project? What do I have to do to comply with RST's license?__<br/>
+
+- You are free to copy or modify any portion of the RST code
+- You are free to share the software outputs (data files, plots)
+- If you choose to share the binaries for your new software: 
+    - You must also share the source code
+    - Your software must be licensed under GPL (v3 or later)
+
+
+__Who owns the outputs of RST (processed data files, plots)?__<br/>
+RST outputs belong to the end-user who created them. End-users are free to use and share these outputs, provided that they comply with the SuperDARN data policy. This includes respecting embargoes on Discretionary Time and Special Time data, and providing appropriate acknowledgement in publications. The data policy can be found in the [SuperDARN Principle Investigators' Agreement](https://superdarn.ca/piagreement).
+
+
+__I have more questions about the GPL__<br/>
+There's lots of helpful information [here](https://www.gnu.org/licenses/gpl-faq.html).
+
+
+## History of RST licensing
+
+RST was originally developed at the Johns Hopkins University/Applied Physics Laboratory (JHU/APL). Around 2012, JHU/APL granted permission for RST to be re-licensed so that the SuperDARN community could continue maintaining the software collaboratively. This process has caused confusion over the years, since even scientists who are expert programmers may not be familiar with the intricacies of software licensing and copyright.
+
+A major source of confusion around the RST license is whether it was intended to be licensed under the [GNU General Public License](https://www.gnu.org/licenses/gpl-3.0.en.html) (GPL), or the [GNU __Lesser__ General Public License](https://www.gnu.org/licenses/lgpl-3.0.en.html) (LGPL). This confusion has arisen because, at some point, the LGPL license notice was attached to most of the RST source code. The `AstAlg` library is the exception to this, which has been [clearly marked](https://github.com/SuperDARN/rst-archive/blob/rst.3.1/codebase/analysis/src.lib/astalg/astalg.1.2/LICENSE.txt) with a GPL disclaimer since it was first added to the RST in v3.1. Since `AstAlg` is clearly licensed under the GPL, it follows that the whole of the RST should also be licensed under the GPL. It is possible that the LGPL disclaimer text was added to the remaining RST source code in error since it is very similar to the GPL disclaimer text. To add to this confusion, RST was not distributed with any license file for several releases (v3.5 to v4.3 inclusive). The GPL license file was added in [v4.3.1](https://doi.org/10.5281/zenodo.3634732). The license information across the whole package will be corrected in RST4.6.
```

Comment:
Just fixed the last sentence to say "was" instead of "will be"

```suggestion
A major source of confusion around the RST license is whether it was intended to be licensed under the [GNU General Public License](https://www.gnu.org/licenses/gpl-3.0.en.html) (GPL), or the [GNU __Lesser__ General Public License](https://www.gnu.org/licenses/lgpl-3.0.en.html) (LGPL). This confusion has arisen because, at some point, the LGPL license notice was attached to most of the RST source code. The `AstAlg` library is the exception to this, which has been [clearly marked](https://github.com/SuperDARN/rst-archive/blob/rst.3.1/codebase/analysis/src.lib/astalg/astalg.1.2/LICENSE.txt) with a GPL disclaimer since it was first added to the RST in v3.1. Since `AstAlg` is clearly licensed under the GPL, it follows that the whole of the RST should also be licensed under the GPL. It is possible that the LGPL disclaimer text was added to the remaining RST source code in error since it is very similar to the GPL disclaimer text. To add to this confusion, RST was not distributed with any license file for several releases (v3.5 to v4.3 inclusive). The GPL license file was added in [v4.3.1](https://doi.org/10.5281/zenodo.3634732). The license information across the whole package was corrected in RST4.6.
```

Reply:
Woohoo! Feels good to put things in the past.